### PR TITLE
@types/blessed fix declarations

### PR DESCRIPTION
**Pull request by witchu**
fix TCursor.shape
add blessed.log()

Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/chjj/blessed/blob/master/lib/widget.js for blessed.log(), https://github.com/chjj/blessed/blob/master/lib/widgets/screen.js for TCursor.shape
- [x] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

File: types/blessed/index.d.ts [hunk 1]
```diff
@@ -74,7 +74,7 @@ declare namespace Blessed {
                 /**
                  * Shape of the cursor. Can be: block, underline, or line.
                 */
-                shape: boolean;
+                shape: boolean|string;
```
Comment (DanielRosenwasser) [inline types/blessed/index.d.ts]: I can't find `boolean` anywhere in the source or docs. Also, why not make this `"block" | "underline" | "line"`?

File: types/blessed/index.d.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
-// Type definitions for blessed 0.1.5
+// Type definitions for blessed 0.1.6
```
Comment (DanielRosenwasser) [inline types/blessed/index.d.ts]: Can you turn linting on for this file by [adding a `tslint.json`](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/1066f6fb8280aa7bc0553c27ca76739b4666c6bb#lint) and fixing up the header to only include major.minor version numbers?

Something as simple as

```json
{
    "extends": "dtslint/dt.json"
}
```

will work.

**Review by DanielRosenwasser — DISMISSED** [review]

File: types/blessed/index.d.ts (hunk 2 above)
Comment (witchu) [reply types/blessed/index.d.ts]: after add tslint.json and `npm run lint blessed` I get many errors. How should I do?

File: types/blessed/index.d.ts (hunk 1 above)
Comment (witchu) [reply types/blessed/index.d.ts]: i changed them.

File: types/blessed/index.d.ts (hunk 2 above)
Comment (mhegazy) [reply types/blessed/index.d.ts]: you can add exceptions to rules you do not want to fix. and fix ones you can

**Comment by DanielRosenwasser** [comment]
Let's just remove the linting file and get this merged in. Sorry about the delay!

**Comment by witchu** [comment]
@DanielRosenwasser done

**Pull request merged**